client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.jsx

```diff
@@ -144,7 +145,9 @@ const DataFilesProjectMembers = ({
       className: 'project-members__cell',
       Cell: (el) => (
         <>
-          {mode === 'addremove' && el.row.original.access !== 'owner' ? (
+          {mode === 'addremove' &&
+          el.row.original.access !== 'owner' &&
+          ['OWNER', 'ADMIN'].includes(authenticatedUserQuery?.data?.role) ? (
```

Comment:
Not in the scope of this PR, but we should apply this same logic to the "Edit Descriptions" button for `USER`/`GUEST`, and the "+Add" files/folders function for `GUEST`.
<img width="807" alt="Screen Shot 2022-07-08 at 3 18 45 PM" src="https://user-images.githubusercontent.com/20326896/178065755-e13f21b3-3bfc-483d-a462-26d3cff222c0.png">